### ISSUE TITLE
Jackson version up gradations  

### ISSUE THREAD

**Issue by mahmed8003** [bguerout/jongo#279]
Any near future update with new jackson version?
Current version :  <jackson.version>2.4.1</jackson.version>

I want to use library with NinjaFramework, but ninja depends upon latest version of Jackson, If I keep latest version then jongo start to give error. And if I keep the older version then Ninja framework give errors on http json body post request.

**Comment by ctrimble** [comment]
@mahmed8003 I am on 2.5.X with a recent build of master and things are OK.  Are you having issues with 2.6.X or 2.7.X?

**Comment by mahmed8003** [comment]
I fixed Jackson version to 2.5.1 in my project pom, now everything working fine. After debugging issue I found I was missing curly braces in in projection parameters. Error was not due to jackson version. 

By changing the jackson version in my project I was getting different error stack so I thought error was due to jackson version mismatch. But after detailed debugging I found a curly brace missing in projection parameter. I think Jongo can work with higher version of Jackson, I have tested it with version 2.5.1 so far and it is working fine.
